shared-module/onewireio/OneWire.c

```diff
@@ -59,8 +59,10 @@ bool common_hal_onewireio_onewire_reset(onewireio_onewire_obj_t *self) {
     common_hal_mcu_delay_us(70);
     bool value = common_hal_digitalio_digitalinout_get_value(&self->pin);
     common_hal_mcu_delay_us(410);
+    // test if bus returned high (idle) and not stck at low
```

Comment:
```suggestion
    // test if bus returned high (idle) and not stuck at low
```

Reply:
I am surprised `codespell` didn't catch that.